plc4j/core/src/test/java/org/apache/plc4x/java/PlcDriverManagerTest.java

```diff
@@ -26,21 +37,11 @@ Licensed to the Apache Software Foundation (ASF) under one
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
-import java.io.File;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.URLClassLoader;
-
-import static org.hamcrest.core.Is.is;
-import static org.hamcrest.core.IsInstanceOf.instanceOf;
-import static org.hamcrest.core.IsNull.notNullValue;
-import static org.hamcrest.core.IsNull.nullValue;
-import static org.junit.Assert.assertThat;
-
 public class PlcDriverManagerTest {
 
     /**
      * Tries to get the mock plc driver which is part of this testsuite.
+     * @throws PlcException 
```

Comment:
theses doesn't make sense in Unit-Tests. Better just remove on star `*`

Reply:
to make javadoc comment to block comment.

Reply:
I can make that transformation if that's the policy the team wants.  Anyone want to toss in a +1 on that?  (another policy to document in a DEVELOPMENT.md)